readme.md

```diff
@@ -174,6 +174,13 @@ React key prefix (`string`, default: `'h-'`).
 Pass the original hast node as `props.node` to custom React components
 (`boolean`, default: `false`).
 
+###### `options.mapDepricatedTableCellAttrsToInlineCss`
+
+Map deprecated table-related attributes to css styles as recommended by
```

Comment:
This will be the same as the jsdoc above, with the exception that ``(`boolean`, default: `true`)`` *is* included here, right before the first `.`